src/injected/components/details-dialog.tsx

```diff
@@ -167,6 +174,10 @@ export class DetailsDialog extends React.Component<DetailsDialogProps, DetailsDi
             shouldShowInspectButtonMessage: () =>
                 this.props.dialogHandler.shouldShowInspectButtonMessage(this),
             userConfigurationStoreData: this.state.userConfigurationStoreData,
+            hasSecureTargetPage: this.props.dialogHandler.isTargetPageOriginSecure(),
+            shouldShowInsecureOriginPageMessage: () =>
+                this.props.dialogHandler.shouldShowInsecureOriginPageMessage(this),
+            onClickCopyIssueDetailsButtonHelpMessage: this.onClickCopyIssueDetailsButtonHelpMessage,
```

Comment:
I felt a little confused by the name `onClickCopyIssueDetailsButtonHelpMessage` when I read it initially; based on the name, I expected it to be something that would get called when the button's help message was clicked.

Perhaps it would be more clear to just keep the single existing `onClickCopyIssueDetailsButton` prop, and give it handler which does both the existing `this.props.deps.targetPageActionMessageCreator.copyIssueDetailsClicked()` and the new `setState({ showInsecureOriginPageMessage: ... })` operations? I think this would be more consistent with how `DetailsDialogHandler.inspectButtonClickHandler` works.